modules/nf-core/purecn/run/main.nf

```diff
@@ -0,0 +1,111 @@
+    path normal_db
+    val genome
+
+    output:
+    tuple val(meta), path("*.csv")                             , emit: csv
+    tuple val(meta), path("*_variants.csv")                    , emit: variants_csv
+    tuple val(meta), path("*.pdf")                             , emit: pdf
+    tuple val(meta), path("*.rds")                             , emit: rds
+    tuple val(meta), path("*_amplification_pvalues.csv")       , emit: amplification_pvalues_csv
+    tuple val(meta), path("*_chromosomes.pdf")                 , emit: chr_pdf
+    tuple val(meta), path("*_dnacopy.seg")                     , emit: seg
+    tuple val(meta), path("*_genes.csv")                       , emit: genes_csv
+    tuple val(meta), path("*_local_optima.pdf")                , emit: local_optima_pdf
+    tuple val(meta), path("*.log")                             , emit: log
+    tuple val(meta), path("*_loh.vcf")                         , emit: loh_vcf
+    tuple val(meta), path("*_loh.vcf.gz")                      , emit: loh_vcf_gz
+    tuple val(meta), path("*_loh.vcf.gz.tbi")                  , emit: loh_vcf_tbi
+    tuple val(meta), path("*_loh.csv")                         , emit: loh_csv
+    tuple val(meta), path("*_loh-effects-stats.csv")           , emit: loh_stats_csv
+    tuple val(meta), path("*_loh-effects-stats.genes.txt")     , emit: loh_effects_txt
+    tuple val(meta), path("*_loh-effects-stats.html")          , emit: loh_effects_html
+    tuple val(meta), path("*_loh-effects.vcf.gz")              , emit: loh_effects_vcf
+    tuple val(meta), path("*_loh-effects.vcf.gz.tbi")          , emit: loh_effects_tbi
+    tuple val(meta), path("*_loh-summary.yaml")                , emit: loh_summary_yaml
+    tuple val(meta), path("*_loh-effects.csv")                 , emit: loh_effects_csv
+    tuple val(meta), path("*_segmentation.pdf")                , emit: segmentation_pdf
+    tuple val(meta), path("*_sort_coverage_loess.png")         , emit: sort_coverage_loess_png
+    tuple val(meta), path("*_sort_coverage_loess_qc.txt")      , emit: sort_coverage_loess_qc_txt
+    tuple val(meta), path("*_sort_coverage_loess.txt.gz")      , emit: sort_coverage_loess_txt_gz
+    tuple val(meta), path("*_sort_coverage.txt.gz")            , emit: sort_coverage_txt_gz
+    path "versions.yml"                                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '2.4.0' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+
+    """
+    library_path=\$(Rscript -e 'cat(.libPaths(), sep = "\\n")')
+    Rscript "\$library_path"/PureCN/extdata/PureCN.R \\
+        --out ./ \\
+        --tumor ${coverage} \\
+        --sampleid ${prefix} \\
+        --vcf ${vcf} \\
+        --normaldb ${normal_db} \\
+        --intervals ${intervals} \\
```

Comment:
Check if any of these arguments are optional, and if so, they should be crammed into `$args` instead.

Reply:
According to [Purecn best practices](https://www.bioconductor.org/packages/devel/bioc/vignettes/PureCN/inst/doc/Quick.html#43_PureCN) this is the minimal test run.